mappings/net/minecraft/util/math/ChunkSectionPos.mapping

```diff
@@ -73,6 +99,34 @@ CLASS net/minecraft/class_4076 net/minecraft/util/math/ChunkSectionPos
 	METHOD method_22446 stream (Lnet/minecraft/class_1923;I)Ljava/util/stream/Stream;
 		ARG 0 center
 		ARG 1 radius
+	METHOD method_30551 unpackLocalX (S)I
+		COMMENT Gets the local x-coordinate from the given packed local position.
+		COMMENT @see #packLocal
+		ARG 0 packedLocalPos
+	METHOD method_30552 unpackLocalY (S)I
+		COMMENT Gets the local y-coordinate from the given packed local position.
+		COMMENT @see #packLocal
+		ARG 0 packedLocalPos
+	METHOD method_30553 unpackLocalZ (S)I
+		COMMENT Gets the local z-coordinate from the given packed local position.
```

Comment:
```suggestion
		COMMENT Gets the local Z coordinate from the given packed local position.
```

Reply:
Is this actually correct? I couldn't find any resource that spells x-,y-,z-coordinate without the hyphen.